docs/source/getting_started/installation/index.md

```diff
@@ -0,0 +1,51 @@
+## Installing PyMC-Marketing
+
+PyMC-Marketing requires **Python 3.10 or greater**.
+
+Install and activate an environment (e.g. `marketing_env`) with the `pymc-marketing` package from [conda-forge](https://conda-forge.org). It may look something like the following:
+
+```bash
+conda create -c conda-forge -n marketing_env pymc-marketing
+conda activate marketing_env
+```
+
+You can also install the development version of PyMC-Marketing with:
+
+```bash
+pip install git+https://github.com/pymc-labs/pymc-marketing.git
+```
+
+Alternatively, if you use the [GitHub CLI](https://cli.github.com/), then the required command is just
+
+```bash
+gh repo fork pymc-labs/pymc-marketing
+```
```

Comment:
Wrong place for this. Would be with contribution as alternative to git clone

Reply:
Right! Thanks! Fixed inhttps://github.com/pymc-labs/pymc-marketing/pull/1023/commits/e8c6eed87f8310e1aaf470d9b76bba8b8863fcbb